src/state/slices/portfolioSlice/portfolioSliceCommon.ts

```diff
@@ -6,6 +6,7 @@ import type { Nominal } from 'types/common'
 export type PortfolioAccount = {
   /** The asset ids belonging to an account */
   assetIds: AssetId[]
+  hasActivity?: boolean
```

Comment:
Since we initially set it to false in the slice's default state, we can have the property as required instead of optional. Only places this would need to be added is in `nftApi`'s `Object.values(nftsById).forEach(nft => {` (true) and in tests